testsuite/gnat2goto/tests/loop_in_custom_type/test.adb

```diff
@@ -0,0 +1,9 @@
+procedure Test is
+   From : constant := 1;
+   To : constant := 5;
+   subtype My_Type is Integer range From .. To;
+begin
+   for X in My_Type loop
+      pragma Assert (X>4);
+   end loop;
```

Comment:
Is there any value in adding some assertions that we also expect to succeed? For instance, perhaps having another loop that maybe just counts iterations, followed by an assertion outside the loop that asserts the counter value?

Reply:
Done.

Reply:
Thanks :-)